arlo_server/routes.py

```diff
@@ -401,16 +425,13 @@ def update_jurisdictions_file(election_id=None):
     jurisdictions_file = request.files["jurisdictions"]
     jurisdictions_file_string = jurisdictions_file.read().decode("utf-8-sig")
 
-    if election.jurisdictions_file:
-        db.session.delete(election.jurisdictions_file)
-
+    old_jurisdictions_file = election.jurisdictions_file
```

Comment:
any particular reason this gets defined up here? as far as i can read, it only gets used once below

Reply:
Because `election.jurisdictions_file` gets clobbered on the next line.